server/sonar-db-dao/src/main/java/org/sonar/db/ce/CeTaskCharacteristicDto.java

```diff
@@ -20,6 +20,9 @@
 package org.sonar.db.ce;
 
 public class CeTaskCharacteristicDto {
+
+  public final static String INCREMENTAL_KEY = "incremental";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AModifiersOrderCheck)